pytest-embedded-qemu/tests/test_qemu.py

```diff
@@ -33,6 +33,32 @@ def test_pexpect_by_qemu(dut):
     result.assert_outcomes(passed=1)
 
 
+@qemu_bin_required
+def test_pexpect_make_restart_by_qemu_xtensa(testdir):
+    testdir.makepyfile("""
+        import pexpect
+        import pytest
+
+        def test_pexpect_by_qemu(dut):
+            dut.expect('Hello world!')
+            dut.qemu._hard_reset()
```

Comment:
```suggestion
            dut.hard_reset()
```

same as the following ones

Reply:
Thanks!